pyoverkiz/models.py

```diff
@@ -274,6 +274,54 @@ def __init__(
         self.value = value
         self.type = DataType(type)
 
+    @property
+    def value_as_int(self) -> int | None:
+        if self.type == DataType.NONE:
+            return None
+        if self.type == DataType.INTEGER:
+            return cast(int, self.value)
+        raise TypeError(f"{self.name} is not an integer")
+
+    @property
+    def value_as_float(self) -> float | None:
+        if self.type == DataType.NONE:
+            return None
+        if self.type == DataType.FLOAT:
+            return cast(float, self.value)
+        raise TypeError(f"{self.name} is not a float")
+
+    @property
+    def value_as_bool(self) -> bool | None:
+        if self.type == DataType.NONE:
+            return None
+        if self.type == DataType.BOOLEAN:
+            return cast(bool, self.value)
+        raise TypeError(f"{self.name} is not a boolean")
+
+    @property
+    def value_as_str(self) -> str | None:
+        if self.type == DataType.NONE:
+            return None
+        if self.type == DataType.STRING:
+            return cast(str, self.value)
+        raise TypeError(f"{self.name} is not a string")
+
+    @property
+    def value_as_dict(self) -> dict[str, Any] | None:
+        if self.type == DataType.NONE:
+            return None
+        if self.type == DataType.JSON_OBJECT:
+            return cast(dict[str, Any], self.value)
+        raise TypeError(f"{self.name} is not a JSON object")
+
+    @property
+    def value_as_list(self) -> list[Any] | None:
+        if self.type == DataType.NONE:
+            return None
+        if self.type == DataType.JSON_ARRAY:
+            return cast(list[Any], self.value)
```

Comment:
I was expecting to cast to `list[dict[str, Any]]`, but in some cases we have a list of string.

Reply:
I think it is mainly a list of strings indeed.